CryptoTechReminderSystem.Test/TestDouble/ClockStub.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using CryptoTechReminderSystem.UseCase;
+
+namespace CryptoTechReminderSystem.Test.TestDouble
+{
+    public class ClockStub : IClock
+    {
+        private DateTimeOffset _currentDateTime;
+
+        public ClockStub(DateTimeOffset dateTime)
+        {
+            _currentDateTime = dateTime;
+        }
+
+        public DateTimeOffset Now()
+        {
+            return _currentDateTime;
+        }
+
+        public void AddSpecifiedMinutes(int minutes)
```

Comment:
Is this used?

Reply:
I dont think it is because we're passing in a new clock every time.

Reply:
No it isn't used, we've removed this in https://github.com/chubberlisk/crypto_tech/pull/17/commits/0428589bcade7a4cb6da87297773e3ef77d6c9ac.